packages/swipe-carousel/README.md

```diff
@@ -6,4 +6,135 @@
 npm i @pokeberry/swipe-carousel
 ```
 
-Docs coming soon...
+`swipe-carousel` is a mobile-focused carousel package that is implemented using CSS `scroll-snap` APIs.
+
+It contains three components. A simple set up may look like this:
+
+```tsx
+import {
+  SwipeCarousel,
+  SwipeCarouselItem,
+  SwipeCarouselIndicator,
+  SwipeCarouselRef,
+} from '@pokeberry/swipe-carousel';
+// remember to import the styles that are included
+// style.scss is also available to import
+import '@pokeberry/swipe-carousel/dist/style.css';
```

Comment:
I think we can improve this some way to not dive into dist folder?

Reply:
i don't think we can? this is even how the big libraries do it. like `react-bootstrap` has you do this (from their docs):
```
import 'bootstrap/dist/css/bootstrap.min.css';
```